synapse/storage/data_stores/main/push_rule.py

```diff
@@ -41,6 +41,7 @@ def _load_rules(rawrules, enabled_map):
         rule = dict(rawrule)
         rule["conditions"] = json.loads(rawrule["conditions"])
         rule["actions"] = json.loads(rawrule["actions"])
+        rule["default"] = False
```

Comment:
The spec defines this as [Whether this is a default rule, or has been set explicitly](https://matrix.org/docs/spec/client_server/unstable#get-matrix-client-r0-pushrules).

If I understand the code correctly, push rules are only entered into the DB if they're explicitly set by a user, so indeed this should always be `False`.